README.md

```diff
@@ -3582,25 +3582,77 @@ mysql> SHOW FULL TABLES FROM ps;
 ...
 ```
 
-#### ps_setup_disable_background_threads
```

Comment:
Why have you replaced all this content, rather than just adding to it? Please add back all of the ps_setup_disable_background_threads docs.